source/_integrations/zwave_js.markdown

```diff
@@ -44,18 +44,48 @@ ha_integration_type: hub
 ha_zeroconf: true
 ---
 
-This integration allows you to control a Z-Wave network via the [Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver.
+The Z-Wave integration allows you to control a Z-Wave network via the [Node Z-Wave JS](https://zwave-js.github.io/node-zwave-js/#/) driver.
 
-To Run Z-Wave you will need a [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules), a running Z-Wave JS server (using only **one** of the add-ons or installation methods described below), and the Z-Wave integration.
+## Overview
 
-{% include integrations/config_flow.md %}
+The Z-Wave integration in Home Assistant talks to one of two possible add-ons: the official Z-Wave JS add-on, or the community add-on Z-Wave JS UI. Both add-ons include the Node Z-Wave JS driver. This driver talks to a controller device (Z-Wave dongle or GPIO (general purpose input/output) board)
```

Comment:
It reads a bit weird that we say here that the integration talks to one of two add-ons, and then further below under the advanced instructions we say that you can also do without any add-on.

I think we should not talk about add-ons at all in the overview. I think we should just list the three requirements, a controller, the Z-Wave JS server and the Z-Wave integration.

Then in the next section we explain that the easiest way to get started running the server is to use the official add-on. The other ways of running the server are explained in the advanced installation instructions.

Reply:
Thank you for the input, @MartinHjelmare! I felt the need to introduce some of the terminology around Z-Wave, as I was confused about it during the installation procedure. But I realize that it is a lot of implementation detail to start with. I removed the section.